### ISSUE TITLE
[FEATURE] unable to unblock apps in work profile

### ISSUE THREAD

**Issue by Heclalava** [ukanth/afwall#1243]
**Is your feature request related to a problem? Please describe.**
I'm trying to use Island to sandbox some apps on my device, which uses the native Android work profile. If I install an app into the island (sandbox) AFwall+ is unable to see the app and hence I can't give the sandboxed app internet access. The only way to give the app internet access is to disable the firewall. 

**Describe the solution you'd like**
I would like to see the apps in work profile within the AFWall+ list to be able to assign IP table rules to them.

**Describe alternatives you've considered**
Disable and stop using AFWall+ and use Island instead as a privacy and security app.

**Additional context**
I'm considering a possible solution, should AFWall+ be cloned into the work profile as well to be about to give access to or block apps sandboxed in the work profile?

**Comment by deadbeef2000** [comment]
Hey there, you just have to go to `Preferences --> Experimental`  and turn on `Dual Apps Support`. Did the trick for me :)

**Comment by Heclalava** [comment]
Yeah I did that initially and it didn't work. I also had to also go to
Magisk manager and set Multiuser mode to Device owner managed. Once I did
that I could start seeing the cloned apps from the work profile in AFWall+.

**Comment by brackenhill-mob** [comment]
@deadbeef2000 @Heclalava Thank you guys - this thread solved it for me.

@ukanth Might this be worth adding to the docs and the 1st post in XDA?

**Comment by theAkito** [comment]
> Might this be worth adding to the docs and the 1st post in XDA?

Absolutely worth it. To be honest, this belongs to the FAQ and/or README of this project... This is crucial.
